_docs/style/typography.md

```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
+![]({{site.url}}/assets/images/style/typography/type-caption.png){:width="635px"}
+{% endfigure %}
 {% endcolumn %}
 </div>
 
-### Caption
+<div class="hxRow" markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Sub-body text
+
+Use sub-body text sparingly, since it is the least readable size. For example use this typography style for the following:
+
+- Labeling iconography
+- Labeling the legend on visualizations
+- Providing tertiary details or cards and tiles
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/typography/type-sub-body.png){:width="635px"}
+{% endfigure %}
+{% endcolumn %}
+</div>
 
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-Captions apply to images and visualizations. They should be used to clarify images and not introduce new, pertinent information.
+### Subdued style
+
+Use the subdued style to allow more important information stand out. For example:
+
+- Use the subdued style for keys in key-value lists to provide more visual prominenance to the value.
+- Use this style for non-critical information in lists, such as abbreviated dates, or as a second line of text in a cell
+
+Parameters for use: 
+
+- This style only applies Gray 750 color to the selected text, and does not affect font size or weight
+- This is not the same as inactive style for links or buttons
+- Use this style with text 14px or taller 
+- Do not subdue text on backgrounds darker than Gray 25 (#fafafa) or it will not pass accessibiity standards
 
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
-{% include type-styles.html htmlexample='<p class="hxCaption">Caption</p>' typeface="Roboto" weight="Light (300)" fontsize="14px / 0.875rem" csscode="Gray 800" hex="#616161" linespace="1.5" %}
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/typography/type-subdued.png){:width="635px"}
+{% endfigure %}
 {% endcolumn %}
+
 </div>
 
-### Link
+## Lists
 
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-Links support styles for hover, active, and disabled states, and inherit size and spacing styles from the text to which they are applied. Helix does not support the visited state.
+### Ordered lists
+
+Ordered lists are usually reserved for instances in which the items must occur in a specific order, such as steps in a procedure, or when keeping count is important, such as a top 10 list.
+
+If the order or the final count of the list items does not matter, don’t number them. Doing so creates confusion.
+
+For more information on when and how to use lists, check out the [Nielsen Norman Group’s article](https://www.nngroup.com/articles/presenting-bulleted-lists/) on this topic.
```

Comment:
How to use lists in general or how to use ordered lists?

Reply:
I'm just going to remove this link. It doesn't seem necessary.